dist/options/options.html

```diff
@@ -223,6 +225,23 @@ <h1>
 
             <textarea id="label-list" rows="16"></textarea>
 
+            <h2>Actions</h2>
+
+            <div class="list-type-options">
+                <div class="list-type">
+                    <input type="checkbox" name="listType" class="action" value="archive" id="archive-checkbox">
```

Comment:
When these checkboxes are selected, can you make them have the same turquoise color as the radio buttons?

If it's not a simple quick change, then let's just leave it as is.